openupgrade_scripts/README.rst

```diff
@@ -13,16 +13,16 @@ Openupgrade Scripts
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2FOpenUpgrade-lightgray.png?logo=github
-    :target: https://github.com/OCA/OpenUpgrade/tree/16.0/openupgrade_scripts
-    :alt: OCA/OpenUpgrade
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fopenupgrade-lightgray.png?logo=github
```

Comment:
Why changing uppercase?

Reply:
Oh I did not pay any attention to the casing when I ran oca-gen-addon-readme. I didn't know it mattered, and I'm still slightly annoyed the project name has to be passed on the command line :stuck_out_tongue:

Reply:
It's for reducing the diff essentially, and yes, you need to introduce it, as it can't be deduced from the folder itself.

Reply:
Couldn't we introduce a .oca-project-name file in the root of the addons directory?

Reply:
That's a good idea, but it should be proposed at maintainer-tools, and maybe harmonize it with ocaport hidden directory (not sure if it's `.oca`), and also bootstrap it through the copier template.